docs/source/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+.. _getting-started-chapter:
+
+###############
+Getting Started
+###############
+
+******************
+Buying our sensors
+******************
+
+In order to use this documentation, you fist need to head over to our online `shop`_ and get yourself one of the following sensors:
```

Comment:
In order to run code found in this documentation, you need to head over... 


(after all, people can read the documentation before buying a sensor, they don't need to sensor to read the doc)

Reply:
Bet you didn't see that typo haha - I did .. :(

Reply:
yeah another typo watcher :)